src/quacc/recipes/espresso/_base.py

```diff
@@ -90,7 +88,7 @@ def run_and_summarize(
         binary=calc.template.binary,
     )
 
-    geom_file = template.outputname if template.binary == "pw" else None
+    geom_file = template.outputname if template and template.binary == "pw" else None
```

Comment:
Could you explain this one to me?

Reply:
We need to ensure that `template` is not `None` before trying to access its attributes (`binary` and `outputname`).

Reply:
Ah, of course.